opensplice_cmake_module/cmake/Modules/FindOpenSplice.cmake

```diff
@@ -64,7 +64,6 @@ if(NOT "${_ospl_home} " STREQUAL " ")
     "${_ospl_home}/include/dcps/C++/SACPP")
   set(OpenSplice_LIBRARIES
     "cmxml"
-    "commonserv"
```

Comment:
Of course it still works with the current LGPL version of OpenSplice (6.4).

Afaik we don't use `commonserv`.